contracts/handlers/ERC721Handler.sol

```diff
@@ -118,11 +112,7 @@ contract ERC721Handler is IDepositExecute, HandlerHelpers, ERC721Safe {
             metaData = bytes(erc721.tokenURI(tokenID));
         }
 
-        if (_burnList[tokenAddress]) {
-            burnERC721(tokenAddress, tokenID);
-        } else {
-            lockERC721(tokenAddress, depositer, address(this), tokenID);
-        }
+        lockERC721(tokenAddress, depositer, address(this), tokenID);
```

Comment:
Update the method description in line 90.

Reply:
Done